src/main/java/ch/njol/skript/expressions/ExprBrewingResults.java

```diff
@@ -0,0 +1,112 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.EventRestrictedSyntax;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.inventory.BrewEvent;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Brewing Results")
+@Description("The result items of an 'on brew complete' event.")
+@Examples({
+	"on brew complete:",
+		"\tset {_results::*} to the brewing results"
+})
+@Since("INSERT VERSION")
+@Events("Brewing Complete")
+public class ExprBrewingResults extends SimpleExpression<ItemStack> implements EventRestrictedSyntax {
+
+	static {
+		Skript.registerExpression(ExprBrewingResults.class, ItemStack.class, ExpressionType.SIMPLE,
+			"[the] brewing results");
+	}
+
+	private boolean delayed;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		delayed = isDelayed.isTrue();
+		return true;
+	}
+
+	@Override
+	public Class<? extends Event>[] supportedEvents() {
+		return CollectionUtils.array(BrewEvent.class);
+	}
+
+	@Override
+	protected ItemStack @Nullable [] get(Event event) {
+		if (!(event instanceof BrewEvent brewEvent))
+			return null;
+		return brewEvent.getResults().toArray(ItemStack[]::new);
+	}
+
+	@Override
+	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
+		if (delayed) {
+			Skript.error("Cannot change the 'brewing results' after the event has passed.");
+			return null;
+		}
+		return switch (mode) {
+			case SET, DELETE, ADD, REMOVE -> CollectionUtils.array(ItemStack[].class);
+			default -> null;
+		};
+	}
+
+	@Override
+	public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
+		if (!(event instanceof BrewEvent brewEvent))
+			return;
+		List<ItemStack> itemStacks = new ArrayList<>();
+		if (delta != null) {
+			for (Object object : delta) {
+				if (object instanceof ItemStack itemStack)
+					itemStacks.add(itemStack);
+			}
+		}
+		if ((mode == ChangeMode.SET || mode == ChangeMode.ADD) && itemStacks.size() > 3)
+			warning("A brewing stand can only contain 3 items; Some items will be ignored.");
```

Comment:
Maybe this check should go after the switch statement and use `results`? We consider a brewing stand that already has three items, but the user attempts to add another.